### PR TITLE
Minor system_info() cleanups

### PR DESCRIPTION
**Pull request by dstansby**
- SunPy > sunpy
- Remove extra space before a colon
- More f-strings
- Add installation path (fixes https://github.com/sunpy/sunpy/issues/5523)

### CODE REVIEW

File: sunpy/util/sysinfo.py [hunk 1]
```diff
@@ -81,13 +81,13 @@ def system_info():
         print(f"OS: Windows {platform.release()} {platform.version()}")
     else:
         print("Unknown OS")
-    for sys_info in ['Arch', 'SunPy']:
-        print('{} : {}'.format(sys_info, sys_prop[sys_info]))
+    for sys_info in ['Arch', 'sunpy']:
+        print(f'{sys_info}: {sys_prop[sys_info]}')
     print()
     print("Required Dependices")
```
Comment (Cadair) [inline sunpy/util/sysinfo.py:87]: I also just noticed this is spelt wrong.

File: sunpy/util/sysinfo.py [hunk 2]
```diff
@@ -81,13 +81,13 @@ def system_info():
...
     print("###################")
     for req in base_reqs:
-        print('{}: {}'.format(req, sys_prop[req]))
+        print(f'{req}: {sys_prop[req]}')
     print()
     print("Optional Dependices")
```
Comment (Cadair) [inline sunpy/util/sysinfo.py:92]: and here

File: sunpy/util/sysinfo.py (hunk 1 above)
Comment (dstansby) [reply sunpy/util/sysinfo.py]: Fixed

**Review by Cadair — APPROVED** [review]

**Comment by Cadair** [comment]
Looks good when the tests pass.

**Review by nabobalis — APPROVED** [review]

**Pull request merged**